Path Resolving going to computer absolute

After updated from version 0.2.0 to 1.0.0 the compiled code changed from

```javascript
var components = 
  (function() {
    var map = {
      './LBtn.vue': $0_0,
'./LCard.vue': $0_1,
'./LIcon.vue': $0_2,

    };
    var req = function req(key) {
      return map[key] || (function() { throw new Error("Cannot find module '" + key + "'.") }());
    };
    req.keys = function() {
      return Object.keys(map);
    };
    return req;
  })();
```

to this

```javascript
var components = 
  (function() {
    var map = {
      'E:/GitHub/ui-land/src/components/LBtn.vue': require_context_module_0_0,
'E:/GitHub/ui-land/src/components/LCard.vue': require_context_module_0_1,
'E:/GitHub/ui-land/src/components/LIcon.vue': require_context_module_0_2,

    };
    var req = function req(key) {
      return map[key] || (function() { throw new Error("Cannot find module '" + key + "'.") }());
    };
    req.keys = function() {
      return Object.keys(map);
    };
    return req;
  })()
;
```

If I roll back to version 0.2.0 it cames back to normal.
The code that is being compiled is:
```javascript
const components = require.context('./components', true, /[a-z0-9]+\.vue$/)
```

I checked the code and seems that it begins after the acorn use in version 1.0.0

--- a/src/helper/resolve-require-code.js
+++ b/src/helper/resolve-require-code.js
@@ -25,7 +25,7 @@ module.exports = function genRequireCode(baseDirname, modules) {
 
     const moduleAbsolutePath = Path.resolve(baseDirname, file).replace(/\\/g, '/');
     importCode += genImportCode(moduleName, moduleAbsolutePath);
-    moduleProps += genPropsCode(moduleAbsolutePath, moduleName);
+    moduleProps += genPropsCode(file, moduleName);
   });
   const requireFnCode = (`
   (function() {

